packages/react/src/components/ClientOnly.tsx

```diff
@@ -0,0 +1,8 @@
+import { type SuspenseProps } from 'react'
+import { useIsClient } from './useIsClient'
+
+export const ClientOnly = ({ children, fallback }: SuspenseProps) => {
+  const isClient = useIsClient()
+
+  return isClient ? <>{children}</> : <>{fallback}</>
+}
```

Comment:
1. Shortly if we can please
2. This is ClientOnly. so make type ClientOnlyProps please

```suggestion
export const ClientOnly = ({ children, fallback }: ClientOnlyProps) => <>{useIsClient() ? children : fallback}</>
```

Reply:
I think most of the time `<ClientOnly/>` be used as the HOC, so it need a `children`.
What do you think about using it as?

```ts
interface ClienOnlyProp {
  children: ReactNode:
  fallback?: ReactNode
}
```

Reply:
Cool! That's good for me